instrumentserver/dashboard/__init__.py

```diff
@@ -0,0 +1,257 @@
+"""
+=====================
+Dashboard and Logger:
+=====================
+
+The dashboard package has 2 different programs in it, a parameter logger and a web dashboard to display that logging.
+They both share a single config file from which it reads different settings and which parameters to log.
+The logger is constantly writing in a csv file the values of the specified parameters while the dashboard reads the csv
+file and displays the information.
+
+The dashboard is capable of creating multiple plots side by side and displaying the specified parameters inside of it.
+This can be used to display separately different categories of parameters like temperature in one,
+and pressure in another one.
+
+Both programs can be run simultaneously using the command 'dashboardlogger' or
+separately using the command 'parameterlogger' and 'dashboard' in the command line. It is important to remember that
+for the logger to work, the instrumentserver should already be running with the instruments already created.
+
+The csv file has 5 different columns:
+    time: The time in which this data point has been logged.
+
+    value: The value of the parameter.
+
+    name: The name of the parameter.
+
+    parameter_path: Full name of the parameter, including all of its submodules, as it appears on the instrument driver.
+
+    address: The address of the instrumentserver where the instrument of this parameters lives.
+
+How to set up config file:
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The file should be located in the dashboard folder inside of the instrumentserver package and should be called:
+'config.py'. The file should have a python dictionary named config, this is the dictionary that is being read.
```

Comment:
this is not good. 
the user should never have to put stuff into the package.

a solution could be to look for a config file in the current working directory (with a default name -- say 'instrumentserver-dashboard-cfg.py') and the user should be able to specify a filepath for one, alternatively.

Reply:
you can import from a specific file using importlib.util -- see for instance: https://codeigo.com/python/import-function-from-file#import-file-from-another-location